lib/punycode.js

```diff
@@ -419,7 +419,7 @@ const punycode = {
 	 * @memberOf punycode
 	 * @type String
 	 */
-	'version': '2.0.0',
+	'version': '2.1.0',
```

Comment:
Is this correct? The PR says this is an upgrade to 2.1.1?

Reply:
I did `npm install punycode@2.1.1` so...uh...mistake in the punycode release? /cc @mathiasbynens

Reply:
That was missed in the 2.1.1 release. The code did not change in that release so it should be fine to land this as is.